survey_goals_and_questions.md

```diff
@@ -0,0 +1,169 @@
+* Which other roles do you collaborate with on MLOps?
+    * ML Engineer
+    * Data Scientist
+    * Software Engineer
+    * Data Engineer
+    * SRE/DevOps
+    * Researcher
+    * Data Analyst
+* Where do those people sit?
+    * Your team
+    * Other teams
+* Rate your organization's maturity [1 to 7 scale]
+    * Data science
+    * MLOps
+    * DevOps 
+    * Software engineering
+
+## [3 mins] Suss out where our members lie on the spectrum of "I know more data science (algorithms, math, etc) to I know more engineering (infrastructure, software, etc)" and the relative importance of the associated skills.
+
+### What do we learn / get by asking this?
+* Double-click into *what* each job title actually means in terms of experience and skills
+* Understand the perceived importance of each skill to success
+* Understand where educational gaps are, and thus, what educational materials we need
+* Help individuals determine where they should invest their personal educational effort
+
+### Questions
+* How many years of experience do you have with the following areas?
+    * Backend engineering (infra, APIs, etc)
+    * SRE/DevOps (reliability, etc)
+    * Data pipelines (ETL, ELT, etc)
+    * Streaming data
+    * Algorithm research & development
+    * Algorithm selection
+    * Frontend engineering
+    * WHAT ELSE??
+* How (un)important is each in order to be successful in __your job__?
+* How (un)important is each in order to be successful in __your overall team's success__?
+# [4 mins] ML last use case
+
+This section asks about "the last use case where MLOps was involved" with the intent of building a baseline of how MLOps actually gets applied.
+
+### What do we learn / get by asking this?
+* Help understand the types of real world ML deployments where MLOps is applied
+
+### Draft questions
+* What is the business use case?
+* Who (roles) is involved with each part of the process
+    1) Define the use case & business requirements
+    2) Experiment with models and features
+    3) Deployment of models and features to production
+    4) Post deployment monitoring
+    [HOW TO MAKE THIS NOT LINEAR?]
+* Length of time for different stages of the process
+* Which tools are used?
+* What data sources are using in training and/or production environments?
+* What is the ML application?
+    * Regression
+	* Classification
+	* Probabilistic classification
+	* Retrieval
+	* Ranking/re-ranking
+* What types of data?
+    * tabular
+    * images
+    * text
+    * video
+    * audio
+    * time series
+    * OR multi-modal
+* What is your model's architecture?
+    * boosted (XGboost)
+    * random forest / ensemble model
+    * linear model, regression, elastic net
+    * encoder<>decoder or recurrent model
+    * convolution model
+    * transformer model
+    * graph type of model
+* Approach to releasing models and features (CI/CD, manual releases, etc)
+* Who is the end user of the prediction(s)?
```

Comment:
can we also ask something like who would you consider client? data scientist, or end user. or maybe that's already clear from the above?